core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,11 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+  .popover {
```

Comment:
My 2 cents: just proceed with the PR as-is, only because #3277 is now focused on changing the color (which means that PR will have to remove parts where it duplicates work on this PR).